packages/modeling/src/operations/booleans/martinez/martinez.js

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
let's maintain the original file name; index.js

Reply:
Agreed